src/config.ts

```diff
@@ -1,13 +1,70 @@
-const Config = {
-    NETWORK: 'main',
+/**
+ * Nimiq Cashlink Configuration Handler
+ * Manages environment configuration for the cashlink generator.
+ * 
+ * Features:
+ * - Load environment variables
+ * - Validate configuration values
+ * - Generate cashlink base URLs
+ * - Type-safe configuration access
+ * 
+ * The configuration handler ensures proper setup of the cashlink generator.
+ */
 
-    TOKEN_LENGTH: 6, // length in characters
-    SECRET_SALT_FILE: './secret/salt',
+import { config } from 'dotenv';
 
-    RPC_HOST: '127.0.0.1',
-    RPC_PORT: 8648,
-};
-Config.CASHLINK_BASE_URL = `https://hub.nimiq${Config.NETWORK === 'main' ? '' : '-testnet'}.com/cashlink/`;
+// Load environment variables
+config();
 
-module.exports = Config;
+/**
+ * Validates and ensures environment variables exist
+ * @param value - Environment variable value
+ * @param name - Environment variable name
+ * @returns Validated string value
+ * @throws If environment variable is not set
+ */
+function assertEnvVar(value: string | undefined, name: string): string {
```

Comment:
Can even consider turning this into a [typescript assertion function](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions), but would lead to more verbose code where it's used below.
```suggestion
function assertEnvVar(value: string | undefined, name: string): asserts value {
```